Gopkg.toml

```diff
@@ -2,11 +2,5 @@
   name = "golang.org/x/text"
   version = "0.3.0"
 
-[[constraint]]
-  name = "k8s.io/client-go"
-  # revision for tag "kubernetes-1.12.3"
-  revision = "d082d5923d3cc0bfbb066ee5fbdea3d0ca79acf8"
```

Comment:
?

Reply:
Reverted, I was getting error with version 1.12.3 as my code required 1.13.1 kubernetes. Fixed it by overriding the version in Gopkg.toml